### PR TITLE
Add google analytics tracking for ebenefits navigation

### PR DESCRIPTION
**Pull request by hinzed1127**
## Description
The startup team wants to track logged in users on VA.gov that click on a link that directs them to eBenefits, so that we can use that metric to track success once we introduce SSO

Some of the instances in which the user should be logged in already were implied by the component. for the other ones, I added in the `isLoggedIn` selector to provide that piece of state

Resolves https://github.com/department-of-veterans-affairs/va-sso/issues/41

### CODE REVIEW

File: src/applications/disability-benefits/526EZ/constants.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-export const E_BENEFITS_URL = 'https://www.ebenefits.va.gov/ebenefits/homepage';
```
Comment (bshyong) [inline src/applications/disability-benefits/526EZ/constants.js]: Double checking that this was an intentional removal

**Review by U-DON — APPROVED** [review]
Looks good. I wonder if it would've been easier for you to implement like some `EBenefitsNavigationLink` that has a built-in `onClick` handler that does the `recordEvent`. Not that I know off the top of my head where it should live, though...

File: src/applications/claims-status/components/NoClaims.jsx [hunk 2]
```diff
@@ -9,7 +10,14 @@ export default function NoClaims() {
       <p>
         This page shows only completed claim applications. If you started a
         claim but haven’t finished it yet, go to{' '}
-        <a href="https://www.ebenefits.va.gov/ebenefits-portal/ebenefits.portal">
+        <a
+          href="https://www.ebenefits.va.gov/ebenefits-portal/ebenefits.portal"
```
[the remark this answers is not in the document]
Comment (hinzed1127) [reply src/applications/claims-status/components/NoClaims.jsx]: good catch, I'll double check and see if there's any reason not to add that in

**Comment by hinzed1127** [comment]
Thanks for the review, @U-DON . I didn't want to do anything overly complicated. if the surface area was bigger I would probably try and generalize to a component that takes `isLoggedIn` and whatever is needed for styles, but I think in its limited capacity it's probably fine as is.

**Pull request merged**